pkg/swift/swift/parse.go

```diff
@@ -0,0 +1,79 @@
+package swift
+
+import (
+	"io"
+	"sort"
+	"strings"
+
+	dio "github.com/deepfactor-io/go-dep-parser/pkg/io"
+	"github.com/deepfactor-io/go-dep-parser/pkg/types"
+	"github.com/deepfactor-io/go-dep-parser/pkg/utils"
+	"github.com/liamg/jfather"
+	"golang.org/x/xerrors"
+)
+
+// Parser is a parser for Package.resolved files
+type Parser struct{}
+
+func NewParser() types.Parser {
```

Comment:
Do we need to update support matrix? One of the potential poc customers was looking for it if I remember correctly